packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -169,15 +183,8 @@ class TimeInput extends React.Component {
         meridiem = this.anteMeridiem;
       }
     }
-    if (this.props.variant === TimeUtil.FORMAT_12_HOUR && TimeUtil.getVariantFromLocale(props) === TimeUtil.FORMAT_24_HOUR) {
-      if (process.env.NODE_ENV !== 'production') {
-        // eslint-disable-next-line no-console
-        console.warn('This locale only uses 24 hour clock. The ante meridiem and post meridiem will not be displayed');
-      }
-      this.anteMeridiem = '';
-      this.postMeridiem = '';
```

Comment:
This previously would clear anteMeridiem and postMeridiem if the locale only supports the 24-HOUR variant. It seems `this.postMeridiem` and `this.postMeridiem` will now always return `AM` and `PM` respectively. Would that have any impact?

Reply:
I saw no impact to that change. When I made this change I looked around for any logic that would flex based on AM/PM being set and found none.